clients/drdisas/test_syntax.template

```diff
@@ -0,0 +1,13 @@
+#if defined(AARCH64)
+ f94017a0   ldr
+ a9be7bfd   stp
+disassembly failed: invalid instruction: not enough bytes: 0x88
```

Comment:
Maybe we should *not* have this extra partial instruction in this test, to catch things like the redzone bug.  The simple test already covers this.

Reply:
Done